### PR TITLE
updates to home and mission-control routing links

### PR DESCRIPTION
**Pull request by Schrese**
# Description
This pull request is to update the aesthetics of the children's pages. The dashboard has the entire div as being clickable (rather than just the text) and the mission-control page has the same for the 'read' section. 
This also adds the new images that we need for checking off whether or not a child has completed a story task. 

Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by JLC6290 — DISMISSED** [review]

File: src/components/pages/MissionControl/Checkbox.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import unchecked_box from '../../../assets/icons/unchecked_box.svg';
+import checked_box from '../../../assets/icons/checked_box.svg';
+
+const Checkbox = props => {
+  console.log(props);
```
Comment (Nick-Ohman) [inline src/components/pages/MissionControl/Checkbox.js]: I know they don't want us to have comments in the code but do they want us to take out the console.logs too?

Comment (bramirez96) [reply src/components/pages/MissionControl/Checkbox.js]: Yeah I think so, you don't really want any client-facing console logs

**Review by bramirez96 — DISMISSED** [review]

File: src/components/pages/MissionControl/Checkbox.js (hunk 1 above)
Comment (Schrese) [reply src/components/pages/MissionControl/Checkbox.js]: Thanks, I totally forgot to remove those

**Review by bramirez96 — APPROVED** [review]
Wow! That's pretty neat! Looks great!

**Review by Nick-Ohman — APPROVED** [review]
Looks great!

**Pull request merged**